.github/workflows/master-health-check.yml

```diff
@@ -0,0 +1,32 @@
+name: Master Health Check
+
+on:
+  schedule:
+    # At minute 0 past hour 0800 and 1600.
+    - cron:  '0 8,16 * * *'
```

Comment:
Why 1600? I'd expect 2000

Reply:
1600 as in 4pm. Do you want it to run at 0800 and 2000 (8pm)? I was just doing it 8 hours apart but we can do 12

Reply:
12 hrs apart if we're running it 2x a day. I'd only do 8hrs apart if we were going 3x a day